x-pack/plugins/security_solution/public/common/components/alerts_viewer/histogram_configs.ts

```diff
@@ -30,4 +31,5 @@ export const histogramConfigs: MatrixHistogramConfigs = {
   stackByOptions: alertsStackByOptions,
   subtitle: undefined,
   title: i18n.ALERTS_GRAPH_TITLE,
+  getLensAttributes: getExternalAlertConfigs,
```

Comment:
nit: Can we have a more consistent naming approach? Why the `getExternalAlertConfigs` is assigned to the property which seems to be a lens attributes - are we using a sub configuration? Maybe then we should assign here only the part of the config which is used for lens?

Reply:
Yes, sorry for the confusion, getExternalAlertConfigs should be renamed to getExternalAlertLensAttributes as it is for getting Lens Attributes only. I've renamed all the Lens attributes and their folder.